client/src/components/LoginForm/SignupContent.jsx

```diff
@@ -3,20 +3,18 @@ import Section from '../Section';
 import EditText from '../EditText';
 import DropDown from '../DropDownOutlined/DropDown';
 
-const years = ['2014', '2015', '2016', '2017', '2018', '2019'];
-const majors = [
-  'Computer Engineering',
-  'Electrical Engineering',
-  'Software Engineering',
-];
+const tempYears = ['2014', '2015', '2016', '2017', '2018', '2019'];
+const tempCurriculum = ['7 semesters', '8 semesters'];
+const tempMajors = ['Computer', 'Electrical', 'Software'];
 
 class SignupContent extends Component {
   render() {
     return (
       <Section className="form" flexDirection="column">
+        <text className="error" id="error-msg" />
         <Section className="subform" flexDirection="row">
           <EditText label="Full Name" /> &nbsp;
```

Comment:
<EditText> component needs an **onChange** function as **props**
If you dont wanna implement it yet, you can just write a dummy function that does nothing